packages/block-library/src/page-list/index.php

```diff
@@ -146,27 +146,37 @@ function nest_pages( $current_level, $children ) {
  * @return string Returns the page list markup.
  */
 function render_block_core_page_list( $attributes, $content, $block ) {
+	global $post;
 	static $block_id = 0;
 	$block_id++;
 
+	$children_only = ( isset( $attributes['childrenOnly'] ) && $attributes['childrenOnly'] );
```

Comment:
Do you mind also adding support for this option in the PageList -> NavigationLinks conversion? This was added in https://github.com/WordPress/gutenberg/pull/30390

Reply:
🤔 Looking at the convert to links, it is only allowed if the parent block is a navigation block - it took me a bit to figure this out because my test case is using the PageList in a widget in a sidebar, I didn't wrap it in a Navigation block.

Reply:
The way the Site Editor work now, I don't think we can do this. When in the editor and you click convert to links it will not have the context of what page you are in - so it will hard code the links in the template, but we want it to use the parent of the page you are browsing.

I created a short video that hopefully explains (with audio)

https://user-images.githubusercontent.com/45363/118560925-1ccd0700-b71f-11eb-8a03-b52043b20e8f.mp4